social_engagement/tests/test_engagement.py

```diff
@@ -471,3 +482,159 @@ def test_no_score(self):
             )
 
             self.assertEqual(get_notifications_count_for_user(self.user.id), 0)
+
+    def test__extract_users_from_results(self):
```

Comment:
@UmanShahzad, I've used dunders for them to indicate that these are tests for private methods, but I guess it might look a bit ambiguous and inconsistent.

Reply:
Ah, didn't notice that's what it was. Yep, it's a bit ambiguous because I don't think it's a common practice, so would be nice to make it consistent.